content/tutorials/single-page-tutorials/create-nft-with-wallet-lib.md

```diff
@@ -0,0 +1,197 @@
+# Create a NFT with wallet library and IPFS
+
+In this tutorial you will upload your desired image to IPFS and create a NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "Setup testnet address and receive funds from faucet" ([IOTA Wiki PR under review](https://github.com/iota-community/iota-wiki/pull/711)). If you haven't completed it, please do so and afterwards come back here. You will use the same folder and development environment as in the mentioned prerequired tutorial, which will allow you to reuse the Stronghold account with funds on it.

```

Comment:
```suggestion
In this tutorial, you will upload your desired image to [IPFS](https://ipfs.tech/) and create an NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "Setup testnet address and receive funds from the faucet" ([IOTA Wiki PR under review](https://github.com/iota-community/iota-wiki/pull/711)). If you haven't completed it, please do so, and afterward, come back here. You will use the same folder and development environment as in the mentioned prerequired tutorial, allowing you to reuse the Stronghold account with funds on it.
```

Reply:
just a reminder to update the link when it's published :)

Reply:
Let's just fix it now. And merge the other PR first

Reply:
Updated